src/app/courses/models/courses.model.ts

```diff
@@ -0,0 +1,7 @@
+export interface Footer {
+  id: number;
+  title: string;
+  creationDate: Date;
+  duration: number;
+  description: string;
+}
```

Comment:
'New line at the end of the file' lint issue.

Reply:
fixed, thanks